Makefile

```
@@ -454,6 +454,13 @@ update-dependencies: ## Uses go get -u to update all the dependencies while hold
 	# Copy everything to vendor directory
 	$(GO) mod vendor
 
+vet: ## Run mattermost go vet specifi checks
+	$(GO) get -u github.com/mattermost/mattermost-govet
+	$(GO) vet -vettool=$(GOPATH)/bin/mattermost-govet -license $(TE_PACKAGES)
```

Tools should not be part of `go.mod`:
```suggestion
	env GO111MODULE=off $(GO) vet -vettool=$(GOPATH)/bin/mattermost-govet -license $(TE_PACKAGES)
```